src/views/Correction.vue

```diff
@@ -331,20 +330,20 @@ export default {
     }
 
     // this is the id of THIS correction filing
-    // if falsy, this is a new correction filing
+    // if 0, this is a new correction filing
     // otherwise it's a draft correction filing
-    this.filingId = +this.$route.params.id // number (may be NaN)
+    this.filingId = +this.$route.params.id || 0 // number
```

Comment:
Don't fully understand explicitly setting to 0 here. My assumption is because filingId is of type Number and but you still want to represent a falsy value if there is no id present?

Reply:
This one is a little different than the other filings in which Filing ID is required. In this case I give it a nice fallback value if it isn't passed in as a param (ie, optional)... and the value 0 is what other code uses to denote that this is a "new filing" (instead of resuming a draft).

One alternative would have been to force the callers to include param id=0 to denote that this is a new correction.

You can also contrast this with correctedFilingId which is required. I could have used 0 for that one too but instead I stayed consistent with the other filings that use NaN.

Reply:
Fair enough, thanks for clarifying!